src/test/rgw/test_multi.py

```diff
@@ -468,6 +468,7 @@ def gen_secret():
 
 def gen_bucket_name():
     global num_buckets
+    global run_prefix
```

Comment:
I don't think this needs to be declared global, since `gen_bucket_name()` isn't modifying `run_prefix`